_data/icons/point.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://bafybeifit3vnnlycri4bypy3c3teedcoujsk6pvuq6f6cnx4lupljplkh4",
```

Comment:
this is a directory and not a file

Reply:
Thanks for clarifying, so should it be updated to https://bafybeifit3vnnlycri4bypy3c3teedcoujsk6pvuq6f6cnx4lupljplkh4.ipfs.w3s.link/icon.png ?

Reply:
@ligi Thanks for clarifying, so should it be updated to https://bafybeifit3vnnlycri4bypy3c3teedcoujsk6pvuq6f6cnx4lupljplkh4.ipfs.w3s.link/icon.png ?

Reply:
This is done